front50-api/src/main/java/com/netflix/spinnaker/front50/api/model/pipeline/Trigger.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.front50.api.model.pipeline;
+
+import javax.annotation.Nullable;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.lang.Object;
+import java.util.Set;
+
+
+abstract class ForwardingMap<K, V> implements Map<K, V> {
```

Comment:
Java files should contain only one class and the class name should match the file name.